packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -643,7 +643,7 @@ name: example
 
 flutter:
   assets:
-    - foo.txt
+    - assets/foo.txt
```

Comment:
I believe this test was incorrect and was erroneously passing prior to this change. This change fixes it.